content/department-processes/team-onboarding/_index.md

```diff
@@ -63,6 +63,9 @@ Daily tech meeting / check-in with tech facilitators and managers at 3:30pm on M
  - Add to Trello boards
  - Add to Datacamp (Data hires)
  - Go through recruit project submission, code review, contributing to the tech department repo and specific product contributions (as relevant)
+  
+#### Team Responsibilities:
+ - [Cohort Leader sheet] (https://docs.google.com/spreadsheets/d/1ZMnFbbqEca1NWbpxLZnYoY9S-usG5tjtM0mAzkArfEQ/edit#gid=0)
```

Comment:
Change "Team Responsibilities" to Cohort Leaders and add a little bit of info below as to what a cohort leader is. i.e. "A cohort leader is the junior staff member that is put in charge of a specific cohort, you can find the allocations and responsibilities [here](google drive link)